script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -564,12 +565,12 @@ You're pissed off, are you?
 -- ……だいたいアンタの目的は俺だろう。ロアを殺したいんなら俺を狙え。吸血鬼に成り下がったアンタに、先輩を責める資格はない」
 Besides, you're after %{i}me%{/i}, aren't you?
  If you want to kill Roa, then here I am.
- You have no right to condemn Senpai after debasing yourself by becoming a vampire!"
+ You have no right to condemn Senpai after having degraded into a vampire!"
```

Comment:
Now that you've degraded into a vampire, you have no right to condemn Senpai!
// Original structure feels a little awkward to me, I think its because of the 'after having'

Reply:
Have a suggestion for the follow up?